### ISSUE TITLE
epub3 fixed layout conversion

### ISSUE THREAD

**Issue by God-damnit-all** [yegon666/CbzMage#28]
I have an epub v3 file that has a fixed layout. The only reader I've found that can even read it at all is [Thorium Reader](https://github.com/edrlab/thorium-reader) which uses the [readium-js](https://github.com/readium/readium-js) library for epub rendering.

Unfortunately there isn't a way to print or convert an epub file in Thorium Reader and it's honestly fairly clunky to use. Trying to convert one with Calibre is a mess and it doesn't look like they're going to be getting any support for it any time soon.

Given the nature of this project, converting things to 1:1 cbz files, I was wondering if you could possibly add support for this.

**Comment by ToofDerling** [comment]
I have working code that converts epub files to cbz, I just need to clean it up and add it to CbzMage. So yeah, I can do that but I won't have time to do it before the end of the month (it's vacation time :)). 

In the meantime it's not super hard to do manually: epub files are actually zip archives. So rename the file to .zip and unpack it, then find the directory with the images (it's named EPUB, OEBPS, or OPS), then create a new zip archive containing only the images and finally rename that archive to .cbz.

**Comment by ToofDerling** [comment]
No, I'm not sure at all as I only had a quick look :)

**Comment by ToofDerling** [comment]
I had time to do a little research, here's one thing you can try: unpack the epub and find the directory with the .xhtml files. Try opening a few of them in a Chromium browser (Chrome, Edge etc) and see if they render okay.

**Comment by ToofDerling** [comment]
Please send the book to rotvel AT gmail DOT com

**Comment by ToofDerling** [comment]
> As an aside, if you know of any advanced OCR tools that can do OCR based off a specific font & font size, let me know.

The only experience I have with OCR is [SubtitleEdit](https://github.com/SubtitleEdit/subtitleedit) which can scan image based movie subtitles. I remember reading about some of the different OCR tools it uses in the documentation. That's all I got.

**Comment by ToofDerling** [comment]
> If you provide an email, I can email it to you for testing purposes. You can delete the comment immediately afterward, I'll see it in my email notifications.

Yeah, I'd like to have a look at it. You should have my email now.

**Comment by God-damnit-all** [comment]
> > If you provide an email, I can email it to you for testing purposes. You can delete the comment immediately afterward, I'll see it in my email notifications.
> 
> Yeah, I'd like to have a look at it. You should have my email now.
> 
> Edit: got the book, thanks.

It's been a couple years, any follow-up to this?
